streamer/src/nonblocking/rate_limiter.rs

```diff
@@ -5,28 +5,28 @@ pub struct RateLimiter {
     /// count of requests in an interval
     pub(crate) count: u64,
 
-    /// Throttle start time
-    throttle_start_instant: Instant,
+    /// Rate limit start time
+    start_instant: Instant,
```

Comment:
I think the name of the struct gives us enough context to avoid adding suffix here